rolepermissions/roles.py

```diff
@@ -49,24 +49,30 @@ def get_name(cls):
     @classmethod
     def assign_role_to_user(cls, user):
         """
-        Deletes all of user's previous roles, and removes all permissions
-        mentioned in their available_permissions property.
+        Assign this role to a user.
 
         :returns: :py:class:`django.contrib.auth.models.Group` The group for the
             new role.
         """
-        from rolepermissions.shortcuts import remove_role
-
-        remove_role(user)
-
-        group, created = Group.objects.get_or_create(name=cls.get_name())
+        group, _created = Group.objects.get_or_create(name=cls.get_name())
         user.groups.add(group)
         permissions_to_add = cls.get_default_true_permissions()
         user.user_permissions.add(*permissions_to_add)
 
         return group
 
     @classmethod
+    def remove_role_from_user(cls, user):
```

Comment:
Correct me if I'm wrong but I believe there's an issue here. Supose the following situation:
Doctor has permission `enter_surgery_room`
Surgeon has permission `enter_surgery_room`

A user is granted both the `Doctor` and `Surgeon` roles.
The `Surgeon` role is then removed from the user. The user will loose the `enter_surgery_room` permisson.

I believe that would not be the expected behavior.